libs/screen/image.ts

```diff
@@ -126,14 +126,25 @@ namespace helpers {
     declare function _blitRow(img: Image, xy: number, from: Image, xh: number): void;
 
     //% shim=ImageMethods::_blit
-    declare function _blit(img: Image, xyDst: number, whDst: number, src: Image, xySrc: number, whSrc: number, transparent: boolean): void;
+    declare function _blit(img: Image, src: Image, args: any[]): void;
 
     function pack(x: number, y: number) {
         return (Math.clamp(-30000, 30000, x | 0) & 0xffff) | (Math.clamp(-30000, 30000, y | 0) << 16)
     }
 
+    const _blitArgs: any[] = [];
```

Comment:
@mmoskal something like this? I'm not having luck compiling for native yet.

Reply:
I thought sim also uses RefCollection but I may be wrong. Cpp looks good!